docs/installation.rst

```diff
@@ -18,21 +18,52 @@ For this we need to install Robusta, and also connect at least one destination (
 Standard Installation
 ------------------------------
 
-1. Download the Helm chart and install Robusta-CLI:
+1.  To configure robusta, the Robusta CLI is required. Choose one of the installation methods below.
 
-.. code-block:: bash
-   :name: cb-helm-and-pip
+.. admonition:: Installation Methods
 
-   helm repo add robusta https://robusta-charts.storage.googleapis.com && helm repo update
-   pip install -U robusta-cli --no-cache
-   
+    .. tab-set::
 
-.. admonition:: Common Errors
-    :class: warning
+        .. tab-item:: PIP
+            :name: pip-cli-tab
+
+            .. code-block:: bash
+                :name: pip
+
+                pip install -U robusta-cli --no-cache
+
+            .. admonition:: Common Errors
+                :class: warning
+
+                * Python 3.7 or higher is required
+                * If you are using a system such as macOS that includes both Python 2 and Python 3, run pip3 instead of pip.
+                * Errors about *tiller* mean you are running Helm 2, not Helm 3
+
+        .. tab-item:: Docker
+            :name: docker-cli-tab
+
+            Robusta CLI and Kubectl are included in the image. Any file generated in the image will be copied to your current directory.
```

Comment:
I think this is too much information
I would just say it required docker to be present on the machine